cypress/e2e/edge-firewall/create-edge-firewall-rule-engine-rate-limit.cy.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable cypress/no-unnecessary-waiting */
+import generateUniqueName from '../../support/utils'
+import selectors from '../../support/selectors'
+
+let firewallName, ruleName
+
+//Flag xfail until the api contract is updated on prod
+describe('Edge Firewall spec', { tags: ['@dev5, @xfail'] }, () => {
```

Comment:
OK, so the test is actually born with xfail? 😅 
```suggestion
describe('Edge Firewall spec', { tags: ['@dev5'] }, () => {
```